net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -408,9 +408,16 @@ mwan3_delete_iface_iptables()
 
 }
 
+mwan3_extra_tables_routes()
+{
+	$IP route list table "$1"
+}
+
 mwan3_get_routes()
 {
-	$IP route list table main | sed -ne "$MWAN3_ROUTE_LINE_EXP" | uniq
+	( $IP route list table main ;
```

Comment:
Can we make the source part a little different? I think it is cryptic

```
$IP route list table main | sed -ne "$MWAN3_ROUTE_LINE_EXP" | uniq
config_list_foreach "globals" "rt_table_lookup" mwan3_extra_tables_routes | sed -ne "$MWAN3_ROUTE_LINE_EXP" | uniq
		
```

Reply:
In the way you suggest the result is likely to contain duplicates, hence I wrapped the call to `grep ... | uniq` around all of it.